packages/babel-traverse/src/hub.js

```diff
@@ -1,6 +1,5 @@
 export default class Hub {
-  constructor(file, options) {
```

Comment:
is it even worth having this file if it's just a file?

Reply:
Yeah we can revisit, I just didn't want to change any of the actual transform infrastructure.